<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SHSecurityDlg</class>
 <widget class="QWidget" name="SHSecurityDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>570</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="m_btnBack">
   <property name="geometry">
    <rect>
     <x>895</x>
     <y>10</y>
     <width>51</width>
     <height>47</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="m_lbTitle">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>193</width>
     <height>47</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>安防</string>
   </property>
  </widget>
  <widget class="QWidget" name="m_wgtLaunch" native="true">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>120</y>
     <width>370</width>
     <height>370</height>
    </rect>
   </property>
   <widget class="QPushButton" name="m_btnCamera">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>170</width>
      <height>170</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="m_btnRoomEnv">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>0</y>
      <width>170</width>
      <height>170</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="m_btnWarn">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>200</y>
      <width>170</width>
      <height>170</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="m_btnSetting">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>200</y>
      <width>170</width>
      <height>170</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="m_wgtRoomEnv" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>941</width>
     <height>511</height>
    </rect>
   </property>
   <widget class="QListWidget" name="m_listwgRoom">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>181</width>
      <height>505</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QScrollArea" name="m_scrollAreaDevice">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>0</y>
      <width>751</width>
      <height>505</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="m_scrollAreaWgDevice">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>749</width>
       <height>503</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
